doc/release-notes-11413.md

```diff
@@ -0,0 +1,10 @@
+Updated or changed RPC
+----------------------
+
+The `bumpfee`, `fundrawtransaction`, `sendmany`, `sendtoaddress`, and `walletcreatefundedpsbt`
```

Comment:
Note that this will also trigger bip125 opt-in.

Reply:
Added node.